src/site/content/en/blog/content-visibility/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'CSS Content Visibility: the most impactful performance boost CSS can give you'
+subhead: TBD
+authors:
+  - vladimirlevin
+  - una
+date: 2020-08-05
+hero: hero.jpg
+alt: Stylized photo of a half-hidden person.
+description: TBD
+tags:
+  - blog
+  - css
+---
+
+The [`content-visibility`](https://drafts.csswg.org/css-contain/#propdef-content-visibility) property might be one of the most impactful new CSS properties for improving page load performance. `content-visibility` enables the user agent to skip an element's rendering work, including layout and painting, until it is needed. If a large portion of your content is off-screen, leveraging the `content-visibility` property allows the browser to skip the rendering working for this content, making the initial user load much faster. Additionally, it allows for faster interactions with the on-screen content. Pretty neat.
+
+`content-visibility` relies on primitives within the [the CSS Containment Spec](http://drafts.csswg.org/css-contain/). So, what exactly is this spec? Let's dive in.
+
+## CSS Containment {: #containment }
+
+ -- Caniuse Image Here: caniuse.com/#feat=css-containment --
```

Comment:
```suggestion
`content-visibility` relies on primitives within the [the CSS Containment Spec](http://drafts.csswg.org/css-contain/). So, what exactly is this spec? Let's dive in.

## Browser compatibility

 -- Caniuse Image Here: caniuse.com/#feat=css-containment --

## CSS Containment {: #containment }

```

I prefer to have the compat info in a separate section so that if someone scans the article it's very easy to find